code/datums/abilities/wizard/blink.dm

```diff
@@ -14,6 +14,9 @@
 		if(!holder)
 			return
 
+		if(is_incapacitated(holder.owner))
```

Comment:
Worth mentioning that this will still allow them to blink while handcuffed. Use ``can_act()`` instead if this is unintentional.

Reply:
Nope, that bit is intentional, I feel like unable to blink while cuffed would be a bit too far.